internal/e2e/lds_test.go

```diff
@@ -73,7 +73,7 @@ func TestNonTLSListener(t *testing.T) {
 				Name:    "ingress_http",
 				Address: envoy.SocketAddress("0.0.0.0", 8080),
 				FilterChains: []listener.FilterChain{
-					filterchain(false, envoy.HTTPConnectionManager("ingress_http", "/dev/stdout")),
+					filterchain(false, envoy.HTTPConnectionManager("ingress_http", "/dev/stdout", "contour", 0, false)),
```

Comment:
Whenever I see test fixtures that include zero values for types, 0, and false, its a sure sign that we need 

a. either a helper
b. the API for the thing we're testing, envoy.HTTPConnectionManager has been stretched too far.

Could you try to dry this up with a helper for the moment, I don't think any of those parameters change apart from the name of the route, `ingress_{http,https}`